resources/assets/js/videos/components/videoScreen.vue

```diff
@@ -513,6 +534,16 @@ export default {
             this.setPaused(true);
             this.resetInteractionMode();
         },
+        emitPreviousFrame() {
+            this.$emit('seek', this.video.currentTime, true);
+            if(!this.seeking)
+                this.showPreviousFrame();
+         },
+        emitNextFrame() {
+            this.$emit('seek', this.video.currentTime, true);
+            if(!this.seeking)
+                this.showNextFrame();
+        },
```

Comment:
This should better be moved to videoPlayback.vue because all the logic is there.

Reply:
Also this seems to work (seeking state is active until the new frame is shown) but I don't really understand how it works. I would expect this to show the seeking state, then finish and wait a while until the new frame is found and finally show the new frame. But the seeking state finishes at the same time the new frame is shown...

If we don't find out why this works, we should at least add a comment why the seemingly unnecessary `$emit('seek')` is there.

Reply:
I'm a bit lost on this one... It seems like if we trigger the previous/next frame several time quickly (by keeping the keys down for example) with emitting this seek event, at some point the timeline freeze (wheel is locked in spinning mode and we can't change time anymore). However I tried calling the seek() method directly from VideoPlayback but then the wheel doesn't appear, I suppose this is because the videoTimeline is in charge of it, catching the seek event ? 
I don't have a solution right now, what do you think is best : showing the wheel at risk of freezing timeline or don't and setting `this.seeking` by hand ?